Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM jupyter/base-notebook:latest
+
+LABEL maintainer="Fabian Brandt-Tumescheit <brandtfa@hu-berlin.de>"
+
+# Install requirements
+USER root
+RUN mkdir /var/lib/apt/lists/partial && \
+    apt-get update && \
+    apt-get install -y  --no-install-recommends cmake git build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER 1000
+RUN pip install --upgrade pip 
+RUN pip install setuptools cython powerlaw sklearn seaborn pandas tabulate matplotlib networkx ipycytoscape
+
+# Create working environment
+# This has to be done as root in order to avoid access denied errors.
+USER root
+RUN mkdir -p ${HOME}/networkit
+COPY . ${HOME}/networkit/
+RUN ln -s ${HOME}/networkit/input ${HOME} && ln -s ${HOME}/networkit/notebooks ${HOME}
+RUN mv ${HOME}/notebooks/test_notebooks.py ${HOME}/networkit && rm -rf ${HOME}/work
```

Comment:
I am not happy with the move either. If we think that the file is distracting, this argument also applies to users who download the repo manually. Hence, in that case, the file should be moved to a different directory altogether (`scripts/`), in a different PR.

Reply:
Agreed - removed the removal.